Lib/fontTools/varLib/designspace.py

```diff
@@ -1,5 +1,6 @@
 """Rudimentary support for loading MutatorMath .designspace files."""
 from __future__ import print_function, division, absolute_import
+import collections
```

Comment:
I don't think it would make any difference.

The designspace_test.py also needs to be modified (that's why CI are failing).